public/app/features/auth-config/AuthDrawer.tsx

```diff
@@ -0,0 +1,104 @@
+import { css } from '@emotion/css';
+import React, { useState } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { getBackendSrv } from '@grafana/runtime';
+import { Button, Drawer, Text, TextLink, Switch, useStyles2 } from '@grafana/ui';
+
+export interface Props {
+  onClose: () => void;
+}
+
+const SETTINGS_URL = '/api/admin/settings';
+
+export const AuthDrawer = ({ onClose }: Props) => {
+  const [isOauthAllowInsecureEmailLookup, setOauthAllowInsecureEmailLookup] = useState(false);
+
+  const getSettings = async () => {
+    try {
+      const response = await getBackendSrv().get(SETTINGS_URL);
+      setOauthAllowInsecureEmailLookup(response.auth.oauth_allow_insecure_email_lookup?.toLowerCase?.() === 'true');
+    } catch (error) {}
+  };
+  const updateSettings = async (property: boolean) => {
+    try {
+      const body = {
+        updates: {
+          auth: {
+            oauth_allow_insecure_email_lookup: '' + property,
+          },
+        },
+      };
+      await getBackendSrv().put(SETTINGS_URL, body);
+    } catch (error) {}
+  };
+
+  const resetButtonOnClick = async () => {
+    try {
+      const body = {
+        removals: {
+          auth: ['oauth_allow_insecure_email_lookup'],
+        },
+      };
+      await getBackendSrv().put(SETTINGS_URL, body);
+      getSettings();
+    } catch (error) {}
+  };
+
+  const oauthAllowInsecureEmailLookupOnChange = async () => {
+    updateSettings(!isOauthAllowInsecureEmailLookup);
+    setOauthAllowInsecureEmailLookup(!isOauthAllowInsecureEmailLookup);
+  };
+
+  const subtitle = (
+    <>
+      Configure auth settings. Find out more in our{' '}
+      <TextLink
+        external={true}
+        href="https://grafana.com/docs/grafana/next/setup-grafana/configure-security/configure-authentication/"
+      >
+        documentation
+      </TextLink>
+      .
+    </>
```

Comment:
I think this description and link don't add much value to the config available in the Auth Drawer. This info is very similar to the desc + link from the [main auth page](https://github.com/grafana/grafana/blob/main/public/app/features/auth-config/AuthProvidersListPage.tsx#L60-L74) (the link is the same).

Maybe you can add [a link](https://grafana.com/docs/grafana/next/setup-grafana/configure-security/configure-authentication/#using-the-same-email-address-to-login-with-different-identity-providers) next to the actual `allow_insecure_email_lookup` setting from below.

Reply:
This drawer section is meant to eventually add more settings from the `[auth]` configuration section.

Please take a look at the original design by @vesnadean. This illustrates the overall look of the drawer with more options.

https://www.figma.com/file/VxdG9sTKLBW8rtxwh05PNr/%F0%9F%94%B5-SSO-configuration-in-Grafana-UI?node-id=1424%3A1318&mode=dev

While I agree that the information seems to be repeated, we can't add a link to documentation for every configuration entry. I changed the link to be more specific to the `Settings` section rather than its index.